### ISSUE TITLE
Bug: Cannot exec JupyterConnect <<EOF invalid expression

### ISSUE THREAD

**Issue by bakeryproducts** [jupyter-vim/jupyter-vim#50]
Hi, thanks for this great project.
Can i get some clarification if i've done things right:
1. Installed jupyter-vim
2. jupyter-console & ; vim t.py
3. On JupyterConnect it was an error at this [line](https://github.com/jupyter-vim/jupyter-vim/blob/494f10d612e050fe461d77f57eae003ed08dd4cc/autoload/jupyter.vim#L35), saying that let bla =<<EOF is invalid expression. 
In comment above it said: 
```python
# See ~/.vim/bundle/jedi-vim/autoload/jedi.vim for initialization routine
```
But in jedi-vim repo there is another type of initialization of `init_lines` variable, (which is list of strings) and it seems like it was that way from the beginning: [commit](https://github.com/davidhalter/jedi-vim/pull/730)

4. I fixed init_lines with that type of annotation, and it all works

Im not much of lua/vim programmer, so dont know if its a bug or im doing something wrong.

My setup :
vim 8.1 with +python3 (vim-gtk from Debian apt)
system python3
jupyter-vim on master

**Comment by f12markovic** [comment]
A can confirm this happening on arch with neovim as well

**Comment by f12markovic** [comment]
Also, everything works just fine on neovim-nightly, so I would assume that this way of initializing variables with =<< operator is just a new feature of vim/neovim. 
It should be noted that newer version of vim(>8.2)/neovim(>0.5) are needed.

**Comment by fuzebox1** [comment]
I've been fighting with getting jupyter-vim set up for months on and off, just giving up and going to the browser every time. Installed nvim-nightly as per your comment and now it finally runs! TYVM!

**Comment by tinmarino** [comment]
Fixed by PR #53
